RFCs/0018-Skip-Default-Adapters.md

```diff
@@ -0,0 +1,49 @@
+# 0018 Skip Default Adapters
+
+## Summary
+This note outlines the proposed changes for adding option for skipping initialization of default adapters and thus skipping default adapters to take part in test discovery/execution.
```

Comment:
Nitpick: change to proposed changes for skipping default adapters so that they don't take part in test discovery and execution.